packages/block-library/src/image/edit.js

```diff
@@ -329,6 +331,11 @@ class ImageEdit extends Component {
 		this.setState( {
 			isEditing: ! this.state.isEditing,
 		} );
+		if ( ! this.state.isEditing ) {
```

Comment:
Typically I'd recommend to use the positive form (not negated form) as the first condition in an `if`, `else`, to improve readability.

Reply:
true that, corrected - using the github UI

Reply:
btw these "using the github UI" are comments to self!